meson.build

```diff
@@ -43,8 +43,10 @@ foreach func : test_funcs
 endforeach
 cfg.set('HAVE_SETXATTR', 
         cc.has_function('setxattr', prefix: '#include <sys/xattr.h>'))
+        
+libiconv = cc.find_library('iconv', required: false)
 cfg.set('HAVE_ICONV', 
-        cc.has_function('iconv', prefix: '#include <iconv.h>'))
+        cc.has_function('iconv', prefix: '#include <iconv.h>') and libiconv.found())
```

Comment:
This shouldn't be necessary. If libiconv doesn't exist, has_function should fail anyway. On the other hand, systems that don't require an external iconv library will stop working.

Reply:
I think adding it will be compatible more system. In my system , libiconv.so.2 only exists in /usr/local/lib/, but not /lib64/ . So ,  
has_function do success but libiconv can be found.